docs/route.md

```diff
@@ -125,94 +125,46 @@ This is a simple property, so you can both read it and write to it.
 
 ##### m.route.Link
 
-This component can create a dynamic routable link:
+This component creates a dynamic routed link. Its essential function is to produce `a` links with local `href`s transformed to take account of the [route prefix](#mrouteprefix).
```

Comment:
It also manages history/SPA routing right?

Reply:
I had such a hard time arguing against this in the first place, and the arguments to keep it were not — as far as I could tell — internally coherent. By my reckoning history listeners ought to account for that logic without implementing any more magic on the link itself; the problem is that the advanced functionality (alternative element selectors, route params interpolation, navigation options specification) require invoking `m.route.set`. So that behaviour is always invoked, leading to all sorts of weird unexpected behaviours. Very difficult to rationalise.

There are further irreconcilable contradictions, because `m.route.set` `params` depends on non-determinative interplay with the router: one cannot tell at render time how `params` interpolate into `route`, so `href` transformation won’t work as implied, and current implementation doesn’t even proxy this correctly in the event handler. 

This reveals muddy thinking in design that ought to be fixed on the API level rather than trying to adequately explain dissonant features. But for the meantime this corrects major inaccuracies in consumer expectation without attempting to fully justify the current logic (impossible IMO) or touching the source (😮‍💨).